projects/schematics/src/migrations/2_0/constructor-deprecations/data/login-form.component.migration.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  AUTH_SERVICE,
+  SPARTACUS_CORE,
+  LOGIN_FORM_COMPONENT,
+  SPARTACUS_STOREFRONTLIB,
+  GLOBAL_MESSAGE_SERVICE,
+  FORM_BUILDER,
+  ANGULAR_FORMS,
+  AUTH_REDIRECT_SERVICE,
+  WINDOW_REF,
+  ACTIVATED_ROUTE,
+  ANGULAR_ROUTER,
+  CHECKOUT_CONFIG_SERVICE,
+} from '../../../../shared/constants';
+import { ConstructorDeprecation } from '../../../../shared/utils/file-utils';
+
+export const LOGIN_FORM_COMPONENT_MIGRATION: ConstructorDeprecation = {
```

Comment:
Add this migration to `CONSTRUCTOR_DEPRECATION_DATA` array

Reply:
done